Pages/logout.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Rubik:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,900&display=swap"
+      rel="stylesheet"
+    />
+
+    <!-- Component Library -->
+    <link
+      rel="stylesheet"
+      href="https://stackui.netlify.app/css/documentation.css"
+    />
+    <link rel="stylesheet" href="../css/logout.css" />
+    <title>MenStack</title>
+  </head>
+  <body>
+    <!-- Hero starts -->
+    <section id="hero">
+      <span class="overlay"></span>
+      <div class="logout-message">
+        <p class="p">You have been successfully logged out. 🥸</p>
```

Comment:
Why is there a box after logged out? i think the emoji isnt supported. Other than this everything else looks fine
![image](https://user-images.githubusercontent.com/67316487/154853088-6de0adb1-eb34-4f19-9fa7-37c25b3fd77d.png)

Reply:
Oh yea, its seems to be not supported on some browsers. Will remove it then.